internal/brokers/examplebroker.go

```diff
@@ -318,6 +319,7 @@ func (b *exampleBroker) IsAuthorized(ctx context.Context, sessionID, authenticat
 	return access, infoUser, nil
 }
 
+//nolint:unparam // This function needs to respect the Brokerer interface, even though it can not error out.
 func (b *exampleBroker) handleIsAuthorized(ctx context.Context, sessionInfo sessionInfo, authData map[string]string) (access, infoUser string, err error) {
```

Comment:
This is a private method, so it has nothing to doe with the interface, no?
It should then be removed and the linting help on the broker.

Reply:
Oops, you're right. I confused the functions.

Reply:
and you start to understand now why I dislike those kind of function() -> handleFunction() nesting :p

Reply:
:clown_face:

Reply:
I still think that the error on the signature is important (to match the IsAuthorized method), so I just updated the comment and fixed the return typo.

Reply:
Ok, I won’t fight over that one as it’s only some example code which will evolve anyway.